Bottom: 2446cd96e2bbe54a0a62e44997f335e5acd43821
Top:    58615379b141de469c49a86eea0d21f7e95483f1
Author: Robert Berger <robert.berger@ReliableEmbeddedSystems.com>
Date:   2018-12-27 16:32:58 +0000

oldnoconfig was removed from 4.20 kernel - use olddefconfig instead

Signed-off-by: genius user <genius@ReliableEmbeddedSystems.com>


---

diff --git a/meta/classes/kernel.bbclass b/meta/classes/kernel.bbclass
index e04d2fe..d50f4a9 100644
--- a/meta/classes/kernel.bbclass
+++ b/meta/classes/kernel.bbclass
@@ -492,7 +492,7 @@ sysroot_stage_all () {
 	:
 }
 
-KERNEL_CONFIG_COMMAND ?= "oe_runmake_call -C ${S} CC="${KERNEL_CC}" O=${B} oldnoconfig"
+KERNEL_CONFIG_COMMAND ?= "oe_runmake_call -C ${S} CC="${KERNEL_CC}" O=${B} olddefconfig"
 
 python check_oldest_kernel() {
     oldest_kernel = d.getVar('OLDEST_KERNEL')
